nifi-extension-bundles/nifi-snowflake-bundle/nifi-snowflake-services/src/main/java/org/apache/nifi/snowflake/service/SnowflakeComputingConnectionPool.java

```diff
@@ -190,6 +203,18 @@ protected Collection<ValidationResult> customValidate(final ValidationContext co
         return Collections.emptyList();
     }
 
+    @OnEnabled
+    public void onConfigured(final ConfigurationContext context) throws InitializationException {
+        super.onConfigured(context);
+        oauthService = context.getProperty(TOKEN_PROVIDER).asControllerService(OAuth2AccessTokenProvider.class);
+    }
+
+    private void refreshOauthToken() {
```

Comment:
Recommend adjusting the name:
```suggestion
    private void refreshAccessToken() {
```